context/app/static/js/components/Detail/provenance/ProvGraph/ProvGraph.jsx

```diff
@@ -12,55 +13,63 @@ function ProvGraph(props) {
   const idKey = isOld ? 'hubmap:displayDOI' : 'hubmap:hubmap_id';
   const timeKey = isOld ? 'prov:generatedAtTime' : 'hubmap:created_timestamp';
   const typeKey = isOld ? 'prov:type' : 'hubmap:entity_type';
-  const runRenderProvVis = () =>
-    renderProvVis('prov-vis-react', provData, {
-      getNameForActivity: (id, prov) => {
-        const activity = prov.activity[id];
-        return `${activity['hubmap:creation_action']} - ${activity[idKey]}`;
-      },
-      getNameForEntity: (id, prov) => {
-        const entity = prov.entity[id];
-        // NOTE: The initial entity node was not included in the sample data;
-        // Fallback to ID, if needed. https://github.com/hubmapconsortium/prov-vis/issues/15
-        return entity ? `${entity[typeKey]} - ${entity[idKey]}` : id;
-      },
-      renderDetailPane: (prov) => {
-        const typeEl =
-          typeKey in prov ? (
-            <SectionItem label="Type">
-              <StyledTypography variant="body1">{prov[typeKey]}</StyledTypography>
-            </SectionItem>
-          ) : (
-            <SectionItem label="Type">
-              <StyledTypography variant="body1">{prov['prov:type']}</StyledTypography>
-            </SectionItem>
-          );
-        const idEl =
-          typeKey in prov && ['Donor', 'Sample', 'Dataset'].includes(prov[typeKey]) ? (
-            <SectionItem label="ID" ml>
-              <StyledTypography variant="body1">
-                {/* TODO: Using StyledLink causes "TypeError: props.theme.palette is undefined"
-                    https://github.com/hubmapconsortium/portal-ui/issues/1486 */}
-                <a href={`/browse/${prov[typeKey].toLowerCase()}/${prov['hubmap:uuid']}`}>{prov[idKey]}</a>
-              </StyledTypography>
-            </SectionItem>
-          ) : null;
-        const createdEl =
-          timeKey in prov ? (
-            <SectionItem label="Created" ml>
-              <StyledTypography variant="body1">{prov[timeKey]}</StyledTypography>
-            </SectionItem>
-          ) : null;
-        return (
-          <FlexPaper>
-            {typeEl}
-            {idEl}
-            {createdEl}
-          </FlexPaper>
-        );
-      },
-    });
-  return setTimeout(runRenderProvVis, 0);
+
+  function getNameForActivity(id, prov) {
+    const activity = prov.activity[id];
+    return `${activity['hubmap:creation_action']} - ${activity[idKey]}`;
+  }
+
+  function getNameForEntity(id, prov) {
+    const entity = prov.entity[id];
+    // NOTE: The initial entity node was not included in the sample data;
+    // Fallback to ID, if needed. https://github.com/hubmapconsortium/prov-vis/issues/15
```

Comment:
That repo is archived, so this has fallen off the radar. Could you ...
- Confirm that the referenced entity is still not included in the API response...
- If it's not, ping Bill on Slack and ask if this is something they might fix...
  - if so, make sure there's a tracking issue on their side,
  - if not, make an issue on our side to strip out the useless node before rendering.

Reply:
Filed #1574